src/components/Switch/index.tsx

```diff
@@ -239,16 +266,7 @@ const Switch: VoidFunctionComponent<SwitchProps> = ({
           aria-checked={checked}
           data-variant={variant}
           data-size={size}
-          role="checkbox"
```

Comment:
Can we instead use the `switch` role? https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/switch_role

Reply:
Vu avec Dodo non necessaire